core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -59,7 +59,28 @@ export class OptimizedSsrEngine {
   constructor(
     protected expressEngine: NgExpressEngineInstance,
     protected ssrOptions?: SsrOptimizationOptions
-  ) {}
+  ) {
+    this.logOptions();
+  }
+
+  protected logOptions(): void {
```

Comment:
`query:` thinking out loud - maybe it's worth considering covering such a method by unit tests.  I know that it's only for logging options, nevertheless, its behaviour depends on some conditions and maybe it's worth covering it in terms of any refactoring in the future.